### PR TITLE
Add Funki Sepolia Testnet

### PR DESCRIPTION
**Pull request by hoangtan282**
Introducing Funki Chain, where the magic of having fun online transforms into having fun onchain. Funki Chain reimagines the blockchain experience as an interconnected world brimming with wonder, adventure, and fun.

Thrilled to put Funki on viem — mainnet coming soon!

Funki Testnet
Chain ID: 3397901
RPC: https://funki-testnet.alt.technology/
Faucet: Coming Soon
Explorer: https://sepolia-sandbox.funkichain.com/

---

## PR-Codex overview
This PR adds support for the Funki Sepolia Testnet in the project.

### Detailed summary
- Added `funkiSepolia` definition in `chains/index.ts`
- Defined `funkiSepolia` chain configuration in `chains/definitions/funkiSepolia.ts` including network details, RPC URLs, block explorers, and contracts.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: src/chains/definitions/funkiSepolia.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        address: "0xCeE7ef4dDF482447FE14c605Ea94B37cBE87Ca9D",
+        blockCreated: 5854312,
+      },
+    },
+    l1StandardBridge: {
+      [sourceId]: {
+        address: "0x1ba82f688eF3C5B4363Ff667254ed4DC59E97477",
+        blockCreated: 5854312,
+      },
+    },
```
Comment (jxom) [inline src/chains/definitions/funkiSepolia.ts:42]: These will need to be added here: https://github.com/ethereum-optimism/superchain-registry/tree/110e744c97b4873384ad2da365c281639fc0668e/superchain/extra/addresses/sepolia

**Review by jxom — CHANGES_REQUESTED** [review]
Needs a changeset 

```
bun run changeset
```

File: src/chains/index.ts [hunk 2]
```diff
@@ -88,6 +88,7 @@ export { flareTestnet } from './definitions/flareTestnet.js'
 export { flowPreviewnet } from './definitions/flowPreviewnet.js'
 export { flowMainnet } from './definitions/flowMainnet.js'
 export { flowTestnet } from './definitions/flowTestnet.js'
+export { funkiSepolia } from './definitions/funkiSepolia.js'
```
Comment (jxom) [inline src/chains/index.ts:91]: Alphabetical order please

File: src/chains/definitions/funkiSepolia.ts (hunk 1 above)
Comment (hoangtan282) [reply src/chains/definitions/funkiSepolia.ts]: Removed. We will add them to Superchain later.

**Pull request merged**